packages/components/src/Text/Text.spec.tsx

```diff
@@ -23,4 +23,24 @@ describe("<Text />", () => {
       </p>
     `);
   });
+
+  it("should pass the passthrough ref", async () => {
+    const TestComponent = () => {
+      const textRef = useRef<HTMLParagraphElement>(null);
+
+      useEffect(() => {
+        textRef.current && textRef.current.focus();
+      });
+
+      return (
+        <Text ref={textRef} size="body" className="passthrough" tabIndex={-1}>
+          Some Text
+        </Text>
+      );
+    };
+    render(<TestComponent />);
+    const textElement = await screen.getByText("Some Text");
```

Comment:
`await` only works with `findBy*` queries. We should make this either

```suggestion
    const textElement = await screen.findByText("Some Text");
```

or

```suggestion
    const textElement = screen.getByText("Some Text");
```

We should also definitely get the [testing library eslint plugin](https://github.com/testing-library/eslint-plugin-testing-library) turned on sometime.

Reply:
Added linter in #455